File: src/flags.rs
pub struct Flags {
    pub help:    bool,
    pub nostrip: bool,
    pub reverse: bool,
    pub short:   bool,
}

pub fn standard_flags() -> Flags {
    return Flags{ 
        help:    false, 
        nostrip: false, 
        reverse: false, 
        short:   false,
    }
}

File: flags.go
package main

type flags struct {
	groupby, help, noStrip, reverse bool
	verbose                         uint8
}

func defaultFlags() flags {
	return flags{
		groupby: false,
		help:    false,
		noStrip: false,
		reverse: false,
		verbose: 0,
	}
}
